kratos/python/add_kernel_to_python.cpp

```diff
@@ -123,7 +123,7 @@ void AddKernelToPython(pybind11::module& m)
         .def("InitializeApplication",  [](Kernel& self, KratosApplication& App){ self.Initialize();
         /*RegisterInPythonApplicationVariables(App);*/ }) //&Kernel::InitializeApplication)
         //.def(""A,&Kernel::Initialize)
-        .def("IsImported", &Kernel::IsImported)
+        .def_static("IsImported", &Kernel::IsImported)
```

Comment:
Soooo, this was the reason of the fail. I see

Reply:
:)